src/webm/WebmVorbisDemuxer.ts

```diff
@@ -0,0 +1,20 @@
+import { TransformOptions } from 'stream';
+import { WebmBaseDemuxer } from './WebmBaseDemuxer';
+
+const VORBIS_HEAD = Buffer.from([...'vorbis'].map((x) => x.charCodeAt(0)));
+
+export class WebmVorbisDemuxer extends WebmBaseDemuxer {
+	public _checkHead(data: Buffer) {
+		if (data.readUInt8(0) !== 2 || !data.slice(4, 10).equals(VORBIS_HEAD)) {
+			throw Error('Audio codec is not Vorbis!');
+		}
+
+		this.push(data.slice(3, 3 + data.readUInt8(1)));
+		this.push(data.slice(3 + data.readUInt8(1), 3 + data.readUInt8(1) + data.readUInt8(2)));
+		this.push(data.slice(3 + data.readUInt8(1) + data.readUInt8(2)));
```

Comment:
`readUint8` can be quite expensive to run all the time, as it [checks bounds with each call](https://github.com/nodejs/node/blob/ce4798b55515827f2a68d269d0f433e3ea9828bb/lib/internal/buffer.js#L248-L255).

Instead, store them as a two variables (since you're reading only 2 bits here) and re-use them.